boards/ti/cc1352p1_launchxl/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library()
```

Comment:
Shouldn't you be removing the file also if you plan to do this optimisation for this platform?